src/Moryx.Runtime.Endpoints/Databases/Endpoint/Serialization/DatabaseConfigSerialization.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) 2020, Phoenix Contact GmbH & Co. KG
+// Licensed under the Apache License, Version 2.0
+
+using System;
+using System.Linq;
+using Moryx.Model.Configuration;
+using System.Reflection;
+using Moryx.Serialization;
+using Moryx.Tools;
+using Moryx.Model.Attributes;
+using Moryx.Model;
+
+namespace Moryx.Runtime.Endpoints.Databases.Endpoint.Serialization
+{
+    public class DatabaseConfigSerialization : DefaultSerialization
```

Comment:
I don't really understand this part and the whole `ProssibleConfiguratorsAttribute` but maybe someone else can check that in particular 😅